Modules/System/XmlWriter/src/XmlWriter.Codeunit.al

```diff
@@ -0,0 +1,90 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+// Provides helper functions for System.Xml.XmlWriter
+/// </summary>
+
+codeunit 1385 "XmlWriter"
+{
+    Access = Public;
+
+    var
+        XmlWriterImpl: Codeunit "XmlWriter Impl.";
```

Comment:
I wonder, if we even need an Impl. codeunit for this one? Right now, the facade calls the impl. codeunit with a one-liner. The impl. codeunit the again calls the .NET method with another one-liner - seems a little over-wrapped to me :-)

Reply:
If it's fine for you to do without I will remove it :), I'm just trying to follow the template of the other modules.

Reply:
Actually, the XmlWriterCreateDocument function has four lines now..